docs/source/partial-transformers/partial-transformers.rst

```diff
@@ -166,6 +166,54 @@ You can expect that basic functionality of chimney's ``Transformer`` either work
 fashion in ``PartialTransformer``\s, or have some counterparty methods in the API
 (usually with the `Partial` prefix or suffix in the name).
 
+Preferring total or partial transformer
+---------------------------------------
+
+Under normal circumstances when fields or sealed hierarchy subtypes should be converted, Chimney should be able
+to generate code on its own. If it cannot you could provide it with derivation settings or implicit transformer handling
+problematic fields/subtypes.
+
+But which implicit should be taken in this situation?
```

Comment:
How about non-ambiguous examples, when we have just implicit `Transformer` and just implicit `PartialTransformer` in scope?

Reply:
done